archive/r/rust/README.md

```diff
@@ -10,6 +10,7 @@ Welcome to Sample Programs in Rust!
 - [Fizz Buzz in Rust](https://github.com/TheRenegadeCoder/sample-programs/issues/453)
 - [Reverse a String in Rust](https://github.com/jrg94/sample-programs/issues/261)
 - [Roman Numeral Conversion in Rust](https://github.com/TheRenegadeCoder/sample-programs/issues/536)
+- ROT-13 in Rust
```

Comment:
Can you create an issue to link here? See the other links in the file for examples.

Reply:
Hello, I've made an issue similar to the other sample programs (#1118). Do you need me to commit an updated README linking to the issue?

Reply:
Yes, please!

Reply:
Should be done now 👍